### PR TITLE
Add command to create pom transaction - Closes #4973

### LINKED ISSUE #4973: Create PoM transaction create command
### Description
- Create unlock transaction creation command.
- transaction:create -t 15 or transaction:create:pom should take 2 arguments for block headers
`transaction:create -t 15 fee nonce "header1" "header2"`

### Motivation
With the new BFT violation, PoM transaction command is required to create new transaction

### Acceptance Criteria
- Write description and examples. Example should not include complete header.
- Test both transaction:create -t 15 and transaction:create:pom cases

### Related issues
#4915

### PR DESCRIPTION
**Pull request by shuse2**
### What was the problem?

This PR resolves #4973 

### How was it solved?

- Add function to create PoM transaction in commander

### How was it tested?

- Added unit test
- run `./bin/run transaction:create:pom 1 100 "{\"height\": 3, \"version\": 2, \"maxHeightPrevoted\": 30, \"blockSignature\": \"xxx\"}" "{\"height\": 3, \"version\": 2, \"maxHeightPrevoted\": 31, \"blockSignature\": \"yyy\"}"` where header should be real one (it is using valid header in the test}

### CODE REVIEW

File: commander/src/commands/transaction/create/pom.ts [hunk 1]
```diff
@@ -0,0 +1,223 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2019 Lisk Foundation
```
Comment (ManuGowda) [inline commander/src/commands/transaction/create/pom.ts:3]:
```suggestion
 * Copyright © 2020 Lisk Foundation
```

File: commander/src/commands/transaction/create/pom.ts [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+		},
+		{
+			name: 'fee',
+			required: true,
+			description: 'Transaction fee in LSK.',
+		},
+		{
+			name: 'header1',
+			required: true,
+			description: 'Contradicting block header in JSON string.',
```
Comment (ManuGowda) [inline commander/src/commands/transaction/create/pom.ts:118]:
```suggestion
			description: 'Contradicting block header as JSON string.',
```

File: commander/src/commands/transaction/create/pom.ts [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+		},
+		{
+			name: 'header2',
+			required: true,
+			description: 'Contradicting block header in JSON string.',
```
Comment (ManuGowda) [inline commander/src/commands/transaction/create/pom.ts:123]: same

File: commander/src/commands/transaction/create/pom.ts [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+		},
+	];
+	static description = `
+	Creates a transaction which will report misbehavior of delegate by providing contradicting 2 block headers.
```
Comment (ManuGowda) [inline commander/src/commands/transaction/create/pom.ts:127]:
```suggestion
	Creates a transaction which will report misbehavior of delegate by providing 2 contradicting block headers.
```

File: commander/src/commands/transaction/create/pom.ts (hunk 3 above)
Comment (shuse2) [inline commander/src/commands/transaction/create/pom.ts:123]:
```suggestion
			description: 'Contradicting block header as JSON string.',
```

File: elements/lisk-transactions/test/report_misbehavior.spec.ts [hunk 5]
```diff
@@ -0,0 +1,106 @@
+/*
+ * Copyright © 2019 Lisk Foundation
```
Comment (ManuGowda) [inline elements/lisk-transactions/test/report_misbehavior.spec.ts:2]:
```suggestion
 * Copyright © 2020 Lisk Foundation
```

File: elements/lisk-transactions/src/report_misbehavior.ts [hunk 6]
```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2019 Lisk Foundation
```
Comment (ManuGowda) [inline elements/lisk-transactions/src/report_misbehavior.ts:2]:
```suggestion
 * Copyright © 2020 Lisk Foundation
```

File: commander/src/commands/transaction/create/pom.ts [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+import { flags as flagParser } from '@oclif/command';
+
+import BaseCommand from '../../../base';
+import { ValidationError } from '../../../utils/error';
+import { flags as commonFlags } from '../../../utils/flags';
+import { getNetworkIdentifierWithInput } from '../../../utils/network_identifier';
+import { getPassphraseFromPrompt } from '../../../utils/reader';
+
+interface RawHeader {
+	readonly id: string;
```
Comment (mitsuaki-u) [inline commander/src/commands/transaction/create/pom.ts:30]: do we need id in header?

Comment (shuse2) [reply commander/src/commands/transaction/create/pom.ts:30]: ah we don't, will remove

**Pull request merged**